### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by Sunaries**
- Fixed inconsistencies of Karolis Spring and Letas Stream maps.
- Fixed inconsistencies in general.

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9087,8 +9087,8 @@ QUEST_LV_0100_20151001_009086	It seems that Vubbes of Miner's village went deep
 QUEST_LV_0100_20151001_009087	You barely defeated the Vubbes that barged in. Return to Aras and ask him about the circumstances at Miner's village.
 QUEST_LV_0100_20151001_009088	Defeated Panto Totem and Simorph with the shaman dolls as Molly requested. Tell Molly about it. 
 QUEST_LV_0100_20151001_009089	Purified the corrupted area and defeated Carnivore as Molly requested. Tell Molly about it. 
-QUEST_LV_0100_20151001_009090	Restore Carolis Altar
-QUEST_LV_0100_20151001_009091	Gathering the power of the owls
+QUEST_LV_0100_20151001_009090	Recover the Karolis Altar
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0100.tsv]: Recovering (quest name)

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**